src/main/java/me/ryanhamshire/GriefPrevention/ClaimPermission.java

```diff
@@ -38,6 +38,10 @@
     /**
      * ClaimPermission used for basic access.
      */
+    Creatures(Messages.NoCreaturesPermission),
+    /**
+     * ClaimPermission used for attacking mobs.
+     */
```

Comment:
Comment is beneath the enum value, which was also inserted beneath the access trust comment.